konnectd/ui/src/app.css

```diff
@@ -0,0 +1,88 @@
+/* additional css on top of kpop */
+html {
+  font-family: 'Open Sans', sans-serif;
+}
+
+strong {
+  font-weight: 600;
+}
+
+.oc-login-bg {
+  background-image: url(./images/background.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  z-index: 0;
+}
+
+#loader {
+  /* NOTE(longsleep): White here needed because of the background image */
+  color: white;
+  text-shadow: #000 0 0 1px;
+}
+
+.oc-logo {
+  position: absolute;
+  top: -120px;
+  left: 50%;
+  height: 100px;
+  transform: translateX(-50%);
+}
+
+.oc-input {
+  background-color: #042047;
+  border: 1px solid rgba(78,133,200,.8);
+  border-radius: 3px;
+  color: rgba(255,255,255,.8);
+  width: 300px;
+  height: 40px;
+  padding: 16px;
+  box-sizing: border-box;
+  font-size: 1rem;
+}
+
+.oc-input:focus {
+  outline: none;
+}
+
+.oc-input::placeholder {
+  color: rgba(78,133,200,.8);
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line function (function-comma-space-after)](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6337760)